lib/asana/resources/tag.rb

```diff
@@ -125,16 +129,6 @@ def delete()
         client.delete("/tags/#{id}") && true
       end
 
-      # Returns the compact task records for all tasks with the given tag.
-      # Tasks can have more than one tag at a time.
-      #
-      # per_page - [Integer] the number of records to fetch per page.
-      # options - [Hash] the request I/O options.
-      def get_tasks_with_tag(per_page: 20, options: {})
```

Comment:
Oh bother, this is one where we can't use alias_method, because we're having to delegate across resources...